docs/en/guides/pipeline-integrations/administration/config-staging-and-rollout.md

```diff
@@ -6,3 +6,489 @@ lang: en
+
+When a new project is set up, a  _Primary_  configuration is created. This is the configuration that toolkit will use by default for the project. In addition to the primary configuration, you can create additional configurations for a project. These can exist in parallel and are useful if you for example want to privately test some modifications, upgrade some apps or do development without impacting the entire team. Additional configurations are created by a process called  _cloning_, a process where a configuration is copied to a new location.
+
+Once you have cloned your configuration, your setup may look something like this:
+
+![](images/config-staging-and-rollout/pc_overview.png)
+
+In addition to the studio level  `tank`  command and your primary project configuration  `tank`  command, you now have a new pipeline configuration which has its own  `tank`  command. If you run this  `tank`  command, you will operate exclusively on the configuration located in the staging sandbox. So if you want to test out some new things in Maya, you can simply navigate to your cloned sandbox, run  `./tank Shot xyz launch_maya`  and the {% include product %} menu which appears in Maya will reflect the configuration inside of your staging sandbox rather than your Primary configuration.
+
+Note that the studio level  `tank`  command always uses the Primary config, so the only way to access a cloned configuration is by navigating to its location and using the  `tank`  command that is located in that folder. In {% include product %}, you can assign a specific set of users to a pipeline configuration entry, and any users that are associated with a configuration will new see menu entries appear in addition to those coming from the Primary config:
+
+![](images/config-staging-and-rollout/pc_shotgun.png)
+
+### Creating a Cloned Configuration
+
+In order to create a staging sandbox where you can safely test your upgrades, navigate to the pipeline configurations for your project in {% include product %} and right click on the Primary configuration. An option to clone your configuration will appear:
+
+![](images/config-staging-and-rollout/clone_menu.png)
+
+Once you click it, you'll be presented with a dialog where you specify the path on disk where you want this new configuration to be located. If you are running a multi OS setup, make sure to put in paths for all relevant platforms.
+
+![](images/config-staging-and-rollout/clone_dialog.png)
+
+When you press ok, toolkit will copy the configuration across and set up the clone. It will also associate your {% include product %} user with this configuration, so as soon as you have created your clone, you will see new menu items appearing inside {% include product %}.
```

Comment:
```suggestion
When you press ok, Toolkit will copy the configuration across and set up the clone. It will also associate your {% include product %} user with this configuration, so as soon as you have created your clone, you will see new menu items appearing inside {% include product %}.
```